new/LIT7125QeneRufael.xml

```diff
@@ -0,0 +1,76 @@
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BNFabb145"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne possibly of the śǝllāse type according to the classification in 
+                    <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl>, with a long verse at the 
+                    beginning, erroneously divided into two by an incorrect verse marker (።). The type and author are not given in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Wazema"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-10-30">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55vb to 55vc. Division into verses is indicated with ። . 
+                    The end of the poem is indicated with ። ። . One or two characters at the end of each line in column 55vc are not recognizable in the 
+                    digital reproduction due to tight binding..</note>
+                <ab>
+                    <l n="1">ሩፋኤል፡ ላዕከ፡ ተሰብኦ፡ ባሕረ፡ ተሰብኦ፡ አመ፡ ጸበተ። እንዘ፡ ይነሥእ፡ እምቈጽል፡ አርአያ፡ ወእምአኅማር፡ አምሳለ፡ 
+                        <del rend="strikethrough">፩</del>።</l>
+                    <l n="2">ለባሕረ፡ ሥጋዊ፡ ዲቤሁ፡ ጸለለ።</l>
+                    <l n="3">ፍጻሚ፡ ዕመቁሰ፡ ይር<cb n="55vc"/>አይ፡ ፈጣሪ፡ እስመ፡ 
+                        በእንቲአ<unclear>ነ</unclear><supplied resp="CH" reason="undefined">፡ ብ</supplied>ህለ።</l>
+                    <l n="4">ወረደ፡ ታሕተ፡ ወተቈል<supplied reason="undefined" resp="CH">ቈለ።</supplied></l>
```

Comment:
I am sending you separately the transcription of the last lines kindly prepared by Nafisa in Paris (she wrote down the letters which are two close to the gut margine and therefore poorly invisible on the photos).